server/controllers/courses/postApplyCourse.js

```diff
@@ -1,17 +1,17 @@
-const postApplyCourse = require('../../../database/queries/postApplyCourse');
+const postApplyCourse = require('../../database/queries/postApplyCourse');
 
-exports.postCourse = (req, res) => {
+exports.applyCourse = (req, res) => {
   const { id, datesId, courseId } = req.body;
```

Comment:
@AbdallahAmmar96 If you want, I think you could get `id` (or maybe it's courseId) from `req.params` and then the client doesn't need to also put the `id` / `courseId` into the post body, which is a little simpler (and then you don't have to worry about what to do if a client does an invalid call like `PUT /course/1 { courseId: 2 }`)

Reply:
Ok, I take the id from the params